release/cloud_builder.sh

```diff
@@ -62,7 +62,7 @@ done
 
 DEFAULT_GCS_PATH="https://storage.googleapis.com/istio-release/releases/${TAG_NAME}"
```

Comment:
It seems like the "https://storage.googleapis.com/" should also be removed from the DEFAULT_GCS_PATH line.

I assume this change has to do with the "ISTIO_URL" line in the Makefile.  An alternative is to use "ISTIO_URL=$TEST_PATH" rather than "ISTIO_GCS=$TEST_PATH" in this script, but I suppose ISTIO_GCS is more appropriate since the motivation for the ISTIO_URL option was to provide a way for people to be able to do a build without relying on GCS.